### PR TITLE
ID to UUID

### PR DESCRIPTION
**Pull request by kylecarlstrom**

### CODE REVIEW

File: server/quickstart/views.py [hunk 1]
```diff
@@ -109,7 +108,7 @@ class AuthorList(APIView):
     Returns a list of all authors
     """
     def get(self, request, format=None):
-        currentUser = request.user.id
+        currentUser = request.user.author.id
```
Comment (kylecarlstrom) [inline server/quickstart/views.py]: So the way this was being done was super brittle. We should basically only be using user to get the author.

File: src/actions/index.js [hunk 2]
```diff
@@ -113,6 +112,7 @@ export function loadPosts(user) {
 * Action that updates the state to log the user in
 */
 function logIn(user) {
+  print(user)
```
Comment (kylecarlstrom) [inline src/actions/index.js]: Whoops

Comment (kylecarlstrom) [inline src/actions/index.js]: Wasn't even correct lol

File: server/quickstart/models.py [hunk 3]
```diff
@@ -23,18 +23,26 @@
 from django.contrib.auth.models import User
 
 from django.db import models
+import uuid
+from django.utils import timezone
 
 class Author(models.Model):
-    user = models.OneToOneField(User)
+    # https://docs.djangoproject.com/en/1.10/ref/models/fields/#datefield
```
Comment (TianZhiWang) [inline server/quickstart/models.py]: I don't think you need to cite this.

**Pull request merged**